_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -23,28 +23,15 @@ The rule applies to any [keyboard event][]:
  - the method `getModifierState` returns `false`; and
  - is [dispatched][] to an [event target][] within a [HTML document][].
 
-## Expectation 1
+## Expectation
 
 For each test target at least one of the following is true:
- - (**Disable**:) a set of one or more [instruments][instrument] are available to prevent any result of the [keyboard event][] within a 10 minute time span of the [event firing][]; or
- - (**Focus**:) any result of the event is prevented if the [event target][] doesn't have [focus][] within a 10 minute time span of the [event firing][]; or
- - (**Remap**:) a set of one or more [instruments][instrument] are available to prevent any result of the [keyboard event][] when the [keyboard event][] method `getModifierState` returns `false` within a 10 minute time span of the [event firing][].
-
-**Note:** The same [instrument][] can be used to **disable**/**remap** more than one event.
-
-**Note:** Preventing results of the event can be done in multiple ways (e.g. removing the event listener; handling the event in a different manner; ...) but the way in which it is done is not relevant for this rule.
+ - (**Disable**:) There is at least one set of [instruments][instrument], where each [instrument][] is in the same [web page][] of the registered event or can be found in a [clearly labeled location][] from that [web page][]. The set of [instruments][instrument] can be used to prevent [changes to the content][changes in content] of the [web page][] resulting from the [keyboard event][] within a 10 minute time span of the [event firing][]; or
+ - (**Focus**:) If the [event target][] doesn't have [focus][] there are no [changes to the content][changes in content] resulting from the [keyboard event][] within a 10 minute time span of the [event firing][]; or
+ - (**Remap**:) There is at least one set of [instruments][instrument], where each [instrument][] is in the same [web page][] of the registered event or can be found in a [clearly labeled location][] from that [web page][]. The set of [instruments][instrument] can be used to prevent [changes to the content][changes in content] of the [web page][] within a 10 minute time span of the [event firing][] when the [keyboard event][] method `getModifierState` returns `false`;
```

Comment:
I agree that it still doesn't read very well… 😕 
A couple of (different) ideas:

1. Now that we're back to a single expectation with "A or B or C", it might be possible to go back to a composite rule with three atomics ones 😱 Breaking out of the bullets list may leave more room for writing in a less confusing way.

2. Using intermediate definitions can maybe help…
In this case I'm thinking about : "clearly labelled set of instruments (or set of clearly labelled instruments)" = "set of instruments that are all clearly labelled"; "blocked event" = "the event makes no changes in content within 10 minutes of being fired".
Then, the expectation becomes:
    - (**Disable**) There is at least one [clearly labelled set of instruments] to [block the event]; or
    - (**Focus**) The event is [blocked] if the target doesn't have focus; or
    - (**Remap**) There is at least one [clearly labelled set of instruments] to [block the event] when its `getModifierState` is false

    The 3rd one might need a bit polishing to get the intention right, also need to make clear that 1st and 3rd mean "after activating the instruments, the event is forever blocked (until instruments are activated to unblock it)".